.github/workflows/fips-checksums.yml

```diff
@@ -69,6 +69,13 @@ jobs:
       - name: save PR number
         run: echo ${{ github.event.number }} > ./artifact/pr_num
       - name: save artifact
+        if: github.server_url == 'https://github.com'
+        uses: actions/upload-artifact@v4
```

Comment:
```yaml
uses: actions/upload-artifact@${{ github.server_url == 'https://github.com' && 'v4' || 'v3' }}
```

Reply:
Unfortunately, it doesn't :( https://github.com/quarckster/openssl-fork/actions/runs/11783243850/workflow#L102

Reply:
Yeah, I just tried a variant of the same, and came to an error on it as well.  Meh...